pkg/database/connection.go

```diff
@@ -68,19 +70,24 @@ func (dao *DAO) InitializeTables() {
 
 	_, err := dao.pool.Exec(context.Background(), "CREATE SCHEMA IF NOT EXISTS search")
 	checkError(err, "Error creating schema.")
```

Comment:
Changes below resolve sonar's warning about line limit of 120 characters. No changes in logic.